src/lib/UI/Config/Mapper/FieldType/RichText/CustomTag.php

```diff
@@ -77,20 +77,8 @@ public function mapConfig(array $enabledCustomTags)
                 );
             }
 
-            $config[$tagName]['label'] = $this->translator->trans(
-                sprintf('ezrichtext.custom_tags.%s.label', $tagName),
-                [],
-                $this->translationDomain
-            );
-            $config[$tagName]['description'] = $this->translator->trans(
-                sprintf(
-                    'ezrichtext.custom_tags.%s.description',
-                    $tagName
-                ),
-                [],
-                $this->translationDomain
-            );
-
+            $config[$tagName]['label'] = "ezrichtext.custom_tags.{$tagName}.label";
```

Comment:
I think that in most cases we use in Admin UI
```
$config[$tagName] = [
    'label' => "ezrichtext.custom_tags.{$tagName}.label",
    'description' => "ezrichtext.custom_tags.{$tagName}.description",
] 
```

Reply:
Indeed that looks better. Fixed via 1282835. Thanks :)